tests/tests.go

```diff
@@ -240,37 +266,29 @@ func waitForGitLabStart(g *fake.GitLab, errChan <-chan error) error {
 }
 
 func (ctl *controller) waitForHealthz() error {
-	timer := time.NewTimer(1 * time.Second)
-	defer timer.Stop()
-
 	healthzURL := "/healthz"
 
-	for {
-		select {
-		case <-timer.C:
-			return nil
-		default:
-			gURL := fmt.Sprintf("%s%s", ctl.rootURL, healthzURL)
-			req, err := http.NewRequest(http.MethodGet, gURL, nil)
-			if err != nil {
-				return fmt.Errorf("fail to create a new GET request(%q), error: %w", gURL, err)
-			}
+	gURL := fmt.Sprintf("%s%s", ctl.rootURL, healthzURL)
```

Comment:
We may need some retries similar to other timer.Ticker() because the server may be up but healthcheck endpoint isn't ready yet.